docs/source/config.json

```diff
@@ -78,7 +78,7 @@
         "Error Handling":  "/fetching/error-handling",
         "Type Conditions": "/fetching/type-conditions",
         "Custom Scalars": "/custom-scalars",
-        "Automatic Persisted Queries": "/fetching/apqs"
+        "Persisted Queries": "/fetching/persisted-queries"
```

Comment:
Do you need to set up a redirect from the old endpoint?

Reply:
Yea that's a good idea!

Reply:
Handled in https://github.com/apollographql/apollo-ios/pull/3160